spec/features/create_new_collection_spec.rb

```diff
@@ -30,6 +30,9 @@
       expect(page).to have_content(collection_attrs.fetch(:name))
       # The deposit button is not present until the collection is accessioned
       expect(page).not_to have_content('+ Deposit to this collection')
+
+      # We should not see the delete button for this collection since it is not a draft
+      expect(page).not_to have_selector("#remove-collection-#{Collection.last.id}")
```

Comment:
Can you do this by querying on the aria-label rather than the id?  The aria-label is more important to the users.  The id serves no purpose that I can tell.

Reply:
Yes, the id was added solely for the tests.  Do you know what the correct syntax to use for finding by aria-label?  i tried what was described here: https://github.com/teamcapybara/capybara/issues/1528 (setting `Capybara.enable_aria_label = true` and then `find("Delete #{work.title}").click`) but couldn't get it to work to find the delete link)

Reply:
first i would try:
`click_link "Delete #{work.title}"` if that failed `find("[aria-label='Delete #{work.title}']").click`

Reply:
Well, waddya know, adding the capybara config and using click_link worked, not sure why I couldn't get that to work before (cause I did try it)